src/lib/components/layout/FormLayout/FormLayout.jsx

```diff
@@ -56,20 +56,11 @@ export const FormLayout = (props) => {
       ].join(' ')}
       {...hasCustomLabelWidth ? { style: { '--rui-custom-label-width': labelWidth } } : {}}
     >
-      {/*
-        Flatten children to one-dimensional array so we get over React Fragments with the `map()`
-        function.
-      */}
-      {flattenChildren(children).map((child) => {
-        if (!React.isValidElement(child)) {
-          return null;
-        }
-
-        return React.cloneElement(child, {
-          inFormLayout: true,
-          layout: fieldLayout,
-        });
-      })}
+      <FormLayoutContext.Provider
+        value={{ layout: fieldLayout }}
```

Comment:
This should be one liner as only one prop is present. Please check and correct another occurances.

Reply:
Decided during a phone call with @bedrich-schindler that we do not want to deal with these things on PR. Either there should be an eslint for it or it is at the discretion of the developer.